### ISSUE TITLE
AndroidManifest and oppwa 4.12.0

### ISSUE THREAD

**Issue by aalhadhoud** [ahmedelkhyary/hyperpay#9]
Hello, 

I have the oppwa version and trying to use the plugin but I get this error:

```

Attribute activity#com.oppwa.mobile.connect.checkout.dialog.CheckoutActivity@windowSoftInputMode value=(adjustResize) from [oppwa.mobile-4.12.0-release.aar] AndroidManifest.xml:27:13-55
	is also present at [:hyperpay_plugin] AndroidManifest.xml:37:13-52 value=(adjustPan).
	Suggestion: add 'tools:replace="android:windowSoftInputMode"' to <activity> element at AndroidManifest.xml:23:9-27:58 to override.

```

my manifest is the same as the example.

**Comment by ahmedelkhyary** [comment]
Hello bro
We still using this version 4.9 .0
    implementation(name: "oppwa.mobile-4.9.0-release", ext: 'aar')

Next release we will upgrade sdk version to 4.12.0
Now try using this version 4.9

**Comment by ahmedelkhyary** [comment]
@aalhadhoud 
Hi bro 
I hope you are doing well 

We released last version 2.0.0 with upgrade hyperpay sdk version 4.12.0
